swift-sdk/Internal/IterableAPIInternal.swift

```diff
@@ -384,11 +412,14 @@ final class IterableAPIInternal: NSObject, PushTrackerProtocol, AuthProvider {
             disableDeviceForCurrentUser()
         }
         
+        authToken = nil
```

Comment:
No need to do this just for authToken. You can remove this line. We should do symmetric things for `userId`, `email` and `authToken`.